lib/misc/cells.coffee

```diff
@@ -0,0 +1,32 @@
+scopes = require './scopes'
+{values} = require 'underscore-plus'
+
+module.exports =
+
+  getCurrentCell: (ed) ->
+    buffer = ed.getBuffer()
+    start = buffer.getFirstPosition()
+    end = buffer.getEndPosition()
+    regexString = atom.config.get 'julia-client.cellDelimiter'
+
+    unless regexString?
+      return [start, end]
+
+    regex = new RegExp regexString
+    cursor = ed.getCursorBufferPosition()
```

Comment:
Should probably iterate over all cursors in the current editor, just like in `blocks.coffee`.